biorbd_optim/path_conditions.py

```diff
@@ -111,7 +111,7 @@ def __init__(self, biorbd_model, all_generalized_mapping=None, q_mapping=None, q
 
 
 class InitialConditions(PathCondition):
-    def __init__(self, initial_guess=()):
+    def __init__(self, initial_guess=(), init_lineaire = False):
```

Comment:
Since more type of initialization are to come in the future and they are by definition mutually exclusive, I think it would make sense to use an Enum with the default being "constant". So right now, the enum would have to possible values (have a look at the Enum for OdeSolver for an example)

Reply:
First, thank you for your answer.
I saw your example and I will correct it.